CHANGELOG.md

```diff
@@ -7,8 +7,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [1.44.13] - 2024-10-15
```

Comment:
need to fix the change log! Lets make sure to not remove anything and also add a new entry for this change